### ISSUE TITLE
Remaining accessibility issues

### ISSUE THREAD

**Issue by alice** [Polymer/paper-input#48]
- `disabled` state should set `aria-disabled="true"`
- Floating label divs should be `aria-hidden="true"`, since the internal input gets an `aria-label` with the same text
